staff/oriol-escolar/cv/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Oriol Escolar</title>
+</head>
+
+<body>
+    
+
+<h1>Oriol Escolar</h1>
+<img src="" alt="Image sample">
+
+<!-- EDUCATION -->
+<section>
+ 
+<h2><u>Education</u></h2>
+
+<ul>
+
+<li>Superior Grade in Videogame development  /  2016-2018</li>
+<li>Bachillerato  /  2014-2016</li>
+<li>First Certificate</li>
+
+</ul>
+
+</section>
+
+<!-- EXPERIENCE -->
+<section>
+
+<h2><u>Experience</u></h2>
+
+<ul>
+
+<li>CEO & Unity Programmer at Panopticon Games  /  September 2018-Present </li>
+<li>Quality Control Trainee at Ubisoft Barcelona  /  June 2017-January 2018</li>
+
+</ul>
+
+</section>
+
+<!-- SKILLS -->
+<section>
+
+<h2><u>Skills</u></h2>
+
+<ul>
+
+    <li>Unity</li>
+    <li>C#</li>
+
+</ul>
+
+</section>
+
+<!-- Languages -->
+<section>
+
+<h2><u>Languages</u></h2>
```

Comment:
The <u> element was deprecated in HTML 4.01. (the <u> element was used to define underlined text).

Reply:
You are right, removed in the code.